generate.sh

```diff
@@ -1,84 +1,14 @@
 # #!/bin/bash
 set -euo pipefail
 
-output_file="passage.gen.go"
+if [ -z "$1" ]; then
+  echo "Required generator file is missing."
+  exit 1
+fi
 
-# Skip pointer on optional fields
-fields="\
-.components.schemas.CreateUserRequest.properties.email \
-.components.schemas.CreateUserRequest.properties.phone \
-.components.schemas.CreateUserRequest.properties.user_metadata \
-.components.schemas.UpdateUserRequest.properties.email \
-.components.schemas.UpdateUserRequest.properties.phone \
-.components.schemas.UpdateUserRequest.properties.user_metadata \
-.components.schemas.CreateMagicLinkRequest.properties.channel \
-.components.schemas.CreateMagicLinkRequest.properties.email \
-.components.schemas.CreateMagicLinkRequest.properties.language \
-.components.schemas.CreateMagicLinkRequest.properties.magic_link_path \
-.components.schemas.CreateMagicLinkRequest.properties.phone \
-.components.schemas.CreateMagicLinkRequest.properties.redirect_url \
-.components.schemas.CreateMagicLinkRequest.properties.send \
-.components.schemas.CreateMagicLinkRequest.properties.ttl \
-.components.schemas.CreateMagicLinkRequest.properties.type \
-.components.schemas.CreateMagicLinkRequest.properties.user_id \
-.components.schemas.MagicLinkType \
-.components.schemas.MagicLinkChannel \
-.components.schemas.UserInfo.properties.user_metadata \
-.components.schemas.UserInfo.properties.webauthn_types"
-for field in $fields; do
-  jq "$field |= . + {\"x-go-type-skip-optional-pointer\": true}" openapi.json > tmp.json && mv tmp.json openapi.json
-done
-
-# Rename component to avoid name clash with generated struct
-jq ".components.schemas.ListPaginatedUsersResponse |= . + {\"x-go-name\": \"PaginatedUsersResponse\"}" openapi.json > tmp.json && mv tmp.json openapi.json
-
-# JSON string of key-value pairs
-transforms='{
```

Comment:
the type renames are covered by https://github.com/passageidentity/passage-go/blob/777d8958f93e4f4c7404979fa7806b6d3618bbc9/overlay.yml#L7-L42

and the enum renames are covered by https://github.com/passageidentity/passage-go/blob/777d8958f93e4f4c7404979fa7806b6d3618bbc9/overlay.yml#L44-L85